lib/jsonlog/log_writer_test.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package jsonlog
+
+import (
+	"bytes"
+	"encoding/json"
+	"testing"
+)
+
+func TestWriteJsonEntry(t *testing.T) {
+	var buf bytes.Buffer
+	w := NewLogWriter(&buf)
+	msg := "The quick brown fox jumped over the lazy dog\n"
```

Comment:
Special characters? ~!@#$%^&*()_+ Can we have a reader and writer test case that includes some/all of this?